bmx.py

```diff
@@ -0,0 +1,86 @@
+import urllib.request
+import xml.etree.ElementTree as ET
+
+from model import Audio, BmxPlaybackResponse, Stream
+
+TUNEIN_DESCRIBE = "https://opml.radiotime.com/describe.ashx?id=%s"
+TUNEIN_STREAM = "http://opml.radiotime.com/Tune.ashx?id=%s"
+
+
+# TODO:  determine how listen_id is used, if at all
+# TODO:  determine how stream_id is used, if at all
+# TODO:  see if there is a value to varying the timeout values
+def tunein_playback(station_id: str) -> BmxPlaybackResponse:
+    describe_url = TUNEIN_DESCRIBE % station_id
+    contents = urllib.request.urlopen(describe_url).read()
```

Comment:
it's using `urllib.request`, not `requests`? Was that a suggestion on the ET docs?